Dockerfile

```diff
@@ -0,0 +1 @@
+FROM wordpress:5.3
```

Comment:
If we're not running any commands to customise the container we don't need a Dockerfile, can just reference wordpress in compose file, e.g. 

```yml
  wordpress-woocommerce-blocks-dev:
     image: wordpress:5.3
```

Though as we evolve, a custom dockerfile might be useful for setting up CLI or custom permissions etc.

Is this container intended purely for e2e tests or for blocks dev generally?

Reply:
no, for tests only, it was like that when I first started copying from wc core

Reply:
Aha, cool. Maybe we can move the compose setup and related config to a subfolder e.g. `e2e-tests` since it's specific to tests.